Challenges/Arrays/ArrayShift.cs

```diff
@@ -1,10 +1,12 @@
 ﻿using System;
+
 namespace Challenges.Arrays
 {
     public class ArrayShift
     {
         public ArrayShift()
         {
+            insertShiftArray
```

Comment:
You can delete this line and leave the method empty until you get a chance to go back and do this challenge.